.github/workflows/test-ios-simulator.yaml

```diff
@@ -11,8 +11,8 @@ jobs:
     runs-on: macos-latest
     timeout-minutes: 60
     outputs:
-      failure_status: ${{ steps.set_failure.outputs.failure_status }}
-      error_status: ${{ steps.set_error.outputs.error_status }}
+      failure_status: ${{ steps.set_status.outputs.failure_status }}
+      error_status: ${{ steps.set_status.outputs.error_status }}
```

Comment:
Why are you using 2 outputs, instead of just one, let's say it could be named `steps.set_status.outputs.status`?

And you'd set the `status` to a string value whatever you like, and compare it later.

Reply:
There's a matrix and it's better to know that one job is broken (has error output) while other job from matrix has failed test (failure output). Simply, it can give more info.

Reply:
Now if there's error from at least one job (doesn't matter if there's failed test in other job), the slack message will be "there's error". There won't be info that one job has failed test. So maybe I will add more precise message text which will include these two outputs?